react_native_fundamentals/index.md

```diff
@@ -1,7 +1,23 @@
 # Module content
 
-- ToDo
-- ToDo
-- ToDo
-- ToDo
-- ToDo
+1. Introduction 
+    - iOS and Android overview
+    - Types of apps
+    - What is React Native
+    - Expo vs React Native
+    - Alternatives 
+2. Environment setup
+3. Creating a new app
+4. Project structure overview
+5. Core Components
+6. Styles
+    - StyleSheet 
+    - Flexbox
+7. List Views 
+8. Platform Specific Code
+9. Architecture 
+    - Old
+    - New
+10. Code Quality Tools
```

Comment:
Maybe we should add review of tools for debugging RN apps?

Reply:
I think it would be a nice idea to separate this topic into a module - Debugging. As well as RN Architecture seems like a more deep dive module that isn't required in fundamentals. Maybe some slight overview mentioned in introduction would be enough.